crates/op-revm/Cargo.toml

```diff
@@ -37,11 +37,12 @@ anyhow.workspace = true
 rstest.workspace = true
 alloy-sol-types.workspace = true
 sha2.workspace = true
-serde_json = { workspace = true, features = ["alloc"] }
+serde_json = { workspace = true, features = ["alloc", "std"] }
 alloy-primitives.workspace = true
+serde = { workspace = true, features = ["derive"] }
 
 [features]
-default = ["std", "c-kzg", "secp256k1", "portable", "blst"]
+default = ["std", "c-kzg", "secp256k1", "portable", "blst", "serde"]
```

Comment:
serde is not on by default in `revm` so it is better to be consistent and not add it